packages/components/src/components/bal-heading/index.html

```diff
@@ -27,6 +27,7 @@ <h2 class="title is-size-2">Colors</h2>
         <bal-heading color="success" level="h4" space="none">Success</bal-heading>
         <bal-heading color="warning" level="h4" space="none">Warning</bal-heading>
         <bal-heading color="danger" level="h4" space="none">Danger</bal-heading>
+        <bal-heading color="hint" level="h4" space="none">Hint</bal-heading>
```

Comment:
Instead of adding hint we could change info to that hint color, because default is already blue.